app/javascript/components/SearchResultsLayout.tsx

```diff
@@ -48,12 +55,22 @@ const SearchResultsLayout = (props: SearchResultsLayoutProps) => {
         <Facets />
         <SearchBar 
           query={props.params.query}
-          results={props.results} 
-        />
-        <Results 
-          results={props.results} 
-          vertical={props.vertical}
+          results={props.resultsData ? props.resultsData.results : null} 
         />
+        {props.resultsData ? (
+          <Results 
+            results={props.resultsData.results}
+            unboundedResults={props.resultsData.unboundedResults}
+            vertical={props.vertical}
+            totalPages={props.resultsData.totalPages}
+            query={props.params.query}
+          />) : (props.params.query || getCurrentPage() > 1) ? (
```

Comment:
This complicated ternary had to be added due to there being no obvious way to tell if Bing has another page of results.

Reply:
There may be a way to simplify this actually. Will do some testing soon.

Reply:
Good point, I think a comment is needed here to explain the logic. And I looked over the PR again this morning, and there may be a way to simplify this ternary. I'll test it out soon.

I'll get the props ordered.

Reply:
👍 on comment and reordering.